### PR TITLE
merge folder openedge-function-python27 and openedge-function-python36  to one folder openedge-function-python for code reuse

### PR DESCRIPTION
**Pull request by chensheng0**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,8 +9,7 @@ package: \
 	openedge-agent/package.zip \
 	openedge-remote-mqtt/package.zip \
 	openedge-function-manager/package.zip \
-	openedge-function-python27/package.zip \
-	openedge-function-python36/package.zip \
+	openedge-function-python/package27.zip \
```
Comment (sherlockblaze) [inline Makefile]: python36.zip ?

File: Makefile [hunk 2]
```diff
@@ -31,8 +30,8 @@ openedge-remote-mqtt/package.zip:
 openedge-function-manager/package.zip:
 	make -C openedge-function-manager
 
-openedge-function-python27/package.zip:
-	make -C openedge-function-python27
+openedge-function-python/package27.zip:
+	make -C openedge-function-python
 
 openedge-function-python36/package.zip:
```
Comment (sherlockblaze) [inline Makefile]: 建议相同的名字：opened-function-python/package36.zip

File: Makefile [hunk 3]
```diff
@@ -187,10 +184,10 @@ release-manifest:
...
 	./bin/manifest-tool-linux-amd64 --username=$(USERNAME) --password=$(PASSWORD) push from-spec tmp/manifest-function-manager.yml
 	# Push openedge-function-python27 manifest
-	sed "s/__REGISTRY__/$(REGISTRY)/g; s/__NAMESPACE__/$(NAMESPACE)/g; s/__VERSION__/$(VERSION)/g;" openedge-function-python27/manifest.yml.template > tmp/manifest-function-python27.yml
+	sed "s/__REGISTRY__/$(REGISTRY)/g; s/__NAMESPACE__/$(NAMESPACE)/g; s/__VERSION__/$(VERSION)/g;" openedge-function-python/manifest27.yml.template > tmp/manifest-function-python27.yml
 	./bin/manifest-tool-linux-amd64 --username=$(USERNAME) --password=$(PASSWORD) push from-spec tmp/manifest-function-python27.yml
 	# Push openedge-function-python36 manifest
-	sed "s/__REGISTRY__/$(REGISTRY)/g; s/__NAMESPACE__/$(NAMESPACE)/g; s/__VERSION__/$(VERSION)/g;" openedge-function-python36/manifest.yml.template > tmp/manifest-function-python36.yml
+	sed "s/__REGISTRY__/$(REGISTRY)/g; s/__NAMESPACE__/$(NAMESPACE)/g; s/__VERSION__/$(VERSION)/g;" openedge-function-python/manifest36.yml.template > tmp/manifest-function-python36.yml
```
Comment (sherlockblaze) [inline Makefile]: 建议复用一个Manifest

File: Makefile (hunk 1 above)
Comment (chensheng0) [reply Makefile]: This is a little tricky which means package27.zip just use for detecting whether package27.zip and package36.zip exist based on the assumption package27.zip and package36.zip all exist in the same time.

File: Makefile (hunk 3 above)
Comment (chensheng0) [reply Makefile]: 这个地方与根目录下 Makefile 里面要打包的对应，如果改一个里面会比较乱

File: Makefile [hunk 4]
```diff
@@ -9,8 +9,7 @@ package: \
 	openedge-agent/package.zip \
 	openedge-remote-mqtt/package.zip \
 	openedge-function-manager/package.zip \
-	openedge-function-python27/package.zip \
-	openedge-function-python36/package.zip \
+	openedge-function-python/package.zip \
```
Comment (ludanfeng) [inline Makefile]: openedge-function-python/package27.zip \
openedge-function-python/package36.zip \

File: Makefile [hunk 5]
```diff
@@ -31,11 +31,11 @@ openedge-remote-mqtt/package.zip:
 openedge-function-manager/package.zip:
 	make -C openedge-function-manager
 
-openedge-function-python27/package.zip:
-	make -C openedge-function-python27
+openedge-function-python/package27.zip:
+	make -C openedge-function-python make27
```
Comment (ludanfeng) [inline Makefile]: make -C openedge-function-python package27.zip

File: openedge-function-python/Makefile [hunk 6]
```diff
@@ -0,0 +1,37 @@
+TAG?=latest
+PYSRC27=$(shell find . -type f -name '*.py' -not -name '*36*')
+PYSRC36=$(shell find . -type f -name '*.py' -not -name '*27*')
+
+make27: package27.zip 
+
+make36: package36.zip
+
```
Comment (ludanfeng) [inline openedge-function-python/Makefile]: 移除上面两个target

**Pull request merged**